src/include/rados/librados.hpp

```diff
@@ -393,7 +393,7 @@ namespace librados
     void zero(uint64_t off, uint64_t len);
     void rmxattr(const char *name);
     void setxattr(const char *name, const bufferlist& bl);
-    void setxattr(const char *name, const buffer::list&& bl);
+    void setxattr(const char *name, const ceph::buffer::list&& bl);
```

Comment:
this was recently fixed in master - probably where the merge conflict is coming from. rebase and you can drop this commit including the accidental submodule update